CHANGELOG.md

```diff
@@ -3,7 +3,42 @@
 All notable changes to this project will be documented in this file. This
 project adheres to [Semantic Versioning](http://semver.org/).
 
-## Unreleased
+## 0.51.0
+
+This release contains improvements to monitoring and an assortment of fixes and improvements.
+
+### Monitoring
+
+- Surface unauthorized response count from OPA API authz handler ([#3378](https://github.com/open-policy-agent/opa/issues/3378)) authored by @ashutosh-narkar
+- plugins: Surface more decision log errors via status API ([#5637](https://github.com/open-policy-agent/opa/issues/5637)) authored by @ashutosh-narkar
+- plugins/rest: Include truncated HTTP response in logs ([#2961](https://github.com/open-policy-agent/opa/issues/2961)) authored by @ashutosh-narkar reported by @gshively11
+
+### Topdown and Rego
+
+- Wasm: Add native support for `object.union_n` built-in function (authored by @Azanul)
+
+### Fixes
+
+- ast: Relaxing strict-mode check for unused args in else-branching functions. ([#5758](https://github.com/open-policy-agent/opa/issues/5758)) authored by @johanfylling reported by @ethanjli
+- ast: Properly set the reported location of unused variables in strict-mode errors. ([#5662](https://github.com/open-policy-agent/opa/issues/5662)) authored by @boranx
+- fmt: report wrong arity for built-in functions. ([#5646](https://github.com/open-policy-agent/opa/issues/5646)) authored by @Trolloldem
+- topdown: http.send(): Ensuring intra-query caching consistency. ([#5736](https://github.com/open-policy-agent/opa/issues/5736)) authored by @johanfylling
+
+### Docs, Website, and Ecosystem
+
+- Fix typo in documentation (authored by @eternaltyro)
+- Update TLS authentication docs (authored by @charlieegan3)
+- Clarification in docs about checksums of Windows executables (authored by @Ronnie-personal)
+- docs: Small fix to context placement in integration (authored by @craigpastro)
```

Comment:
It doesn't matter, but I'm curious as to why there are no links to the above changes.

Reply:
Some of the generated entries simply didn't pick up on the linked issue. I've attempted to manually correct for those; but some were simply just a "naked" PR. I'll go through these again to see if I've missed any that can be linked to an issue.